therapeutic/models/nh_clinical_patient_observation_therapeutic.py

```diff
@@ -0,0 +1,105 @@
+from openerp import models, api
+
+from openerp.addons.nh_observations import fields as obs_fields
+
+
+class NhClinicalPatientObservationTherapeutic(models.Model):
+    """
+    Model for the therapeutic observation.
+    """
+    _name = 'nh.clinical.patient.observation.therapeutic'
+    _inherit = 'nh.clinical.patient.observation'
+
+    _patient_status_selection = [
+        ('AW', 'Awake'),
+        ('CO', 'Consultation'),
+        ('ED', 'Education'),
+        ('GT', 'Group Therapy'),
+        ('LE', 'Leave'),
+        ('OU', 'Out (With Permission)'),
+        ('MI', 'Missing (Inform S/N)'),
+        ('SL', 'Sleeping'),
+        ('TR', 'Transfer'),
+        ('DI', 'Discharge')
+    ]
+
+    # This field is left out of the form description because its type is not
+    # currently supported. This is a good thing as we do not want to display
+    # this field at the moment. It is maintained for use of future features.
+    patient_level = obs_fields.Many2one(
+        comodel_name='nh.clinical.therapeutic.level',
+        string="Patient Level",
+        help="The therapeutic level assigned to the patient at the time of "
+             "the observation being completed.",
+        necessary=False
+    )
+    patient_status = obs_fields.Selection(
+        selection=_patient_status_selection,
+        string="Patient Status",
+        required=True
+    )
+    location = obs_fields.Char(
+        size=100,
+        necessary=False
+    )
+    areas_of_concern = obs_fields.Char(
+        size=100,
+        string="Areas of Concern",
+        necessary=False
+    )
+    one_to_one_intervention_needed = obs_fields.Boolean(
+        string="Intervention Needed?",
+        necessary=False
+    )
+    other_staff_during_intervention = obs_fields.Char(
+        size=200,
+        string="Other Staff During Intervention",
+        necessary=False
+    )
+    other_notes = obs_fields.Text(
+        string="Other Notes",
+        necessary=False
+    )
+
+    @api.model
+    def create(self, values):
```

Comment:
@moderatemisbehaviour  - I don't understand the use for this method. Probs missing something obvious, but we don't have the therapeutics level anywhere in the actual observation itself. As far as I understand, there is currently no link between the level and the observation itself (or on the form itself at least). Could you clarify this please? :)

Reply:
This is the same level as you mentioned in another comment saying that it shouldn't be added yet. It will be removed along with the field and the test for it.